src/app/security-group/sg-rules/sg-rules.component.ts

```diff
@@ -158,6 +180,20 @@ export class SgRulesComponent {
       });
   }
 
+  public getIcmpTypeTranslationToken(type: number): string {
+    return GetICMPTypeTranslationToken(type);
+  }
+
+  public getIcmpCodeTranslationToken(type: number, code: number): string {
```

Comment:
can we pass it as 
`public getIcmpCodeTranslationToken = GetICMPCodeTranslationToken` 
?